Documentation/nvme-tls-key.txt

```diff
@@ -54,15 +54,73 @@ OPTIONS
 -e::
 --export::
 	Write the key data to the file specified by '--keyfile'
-	or stdou if not present.
+	or stdout if not present.
+
+-r <id>::
+--remove <id>::
+	Remove a key from a keyring.
 
 -v::
 --verbose::
 	Increase the information detail in the output.
 
 EXAMPLES
 --------
-No Examples
+
+* Create a new TLS key and insert it directly into the .nvme keyring:
++
+------------
+# nvme gen-tls-key -i -n hostnqn0 -c subsys0
+NVMeTLSkey-1:01:/b9tVz2OXJVISnoFgrPAygyS86XYJWkAapQeULns6PMpM8wv:
+Inserted TLS key 26b3260e
+------------
+
+* Export previously created key from the kernel keyring and store it into a file
++
+------------
+# nvme tls-key -e -f nvme-tls-keys.txt
+------------
+
+* Export/list all keys from the .nvme keyring using nvme and keyctl
++
+------------
+# nvme tls-key --export
+NVMe0R01 hostnqn0 subsys0 NVMeTLSkey-1:01:/b9tVz2OXJVISnoFgrPAygyS86XYJWkAapQeULns6PMpM8wv:
+
+# keyctl show
+Session Keyring
+ 573249525 --alswrv      0     0  keyring: _ses
+ 353599402 --alswrv      0 65534   \_ keyring: _uid.0
+ 475911922 ---lswrv      0     0   \_ keyring: .nvme
+ 649274894 --als-rv      0     0       \_ psk: NVMe0R01 hostnqn0 subsys0
+------------
+
+* Remove/revoke a key using the description as id and verifying with
+keyctl the operation
++
+------------
+# nvme tls-key --remove "NVMe0R01 hostnqn0 subsys0"
```

Comment:
```diff
- # nvme tls-key --remove "NVMe0R01 hostnqn0 subsys0"
+ # nvme tls-key --remove="NVMe0R01 hostnqn0 subsys0"
```

Reply:
The equal sign is not really needed and the rest of the examples in this file don't use it. So why here?

Reply:
I see. By the way do you think the option description `--keyring=<name>` should be (or is better to be) `--keyring <name>` as not using the equal sign?

Reply:
Ah, okay that's where `=` is from. I don't really care which version we use in the docs, but we should try to keep it consistent. Need to check what the rest of the examples use.